server/test/integration/user.js

```diff
@@ -40,7 +40,7 @@ describe('User Routes', () => {
                 });
         });
 
-        it('Should return duplicate', () => {
+        it.skip('Should return duplicate', () => {
```

Comment:
Each test is independent, so the data is refreshed - you have to write in this test to do something before the test runs (like create a user orr run the post request twice)

This file - https://storage.googleapis.com/hatchways-app.appspot.com/programs/project-course/resources/test-express-example.zip - has some example tests